cl/users/views.py

```diff
@@ -202,6 +204,23 @@ def view_api(request: HttpRequest) -> HttpResponse:
     return render(request, "profile/api.html", {"private": True})
 
 
+@login_required
+@never_cache
+def view_webhooks(request: HttpRequest) -> HttpResponse:
+    user = request.user
+    webhook = user.webhooks.first() if user.webhooks.count() > 0 else Webhook()
+    webhook_form = WebhookForm(request.POST or None, instance=webhook)
+    if webhook_form.is_valid():
+        webhook.user = user
+        webhook_form.save()
+        return HttpResponseRedirect(reverse("view_webhooks"))
```

Comment:
I've never seen this style for a django view that receives a POST. Usually there's an `if POST, then...` and an `else GET, then...`. Is this something you worked out or is it coming from a new style I haven't kept up with?

Reply:
@mlissner I try not to 'innovate' too much with an existing codebase. If it's part of the strategy, then I am happy to change styles, but I usually find something nearby and try to emulate the style. In this case, here was my reference: https://github.com/freelawproject/courtlistener/blob/master/cl/users/views.py#L213

Reply:
No worries. I guess I was the innovator. I'm going to push a more traditional version in a sec.